RedditSharp/Subreddit Information/SubredditImage.cs

```diff
@@ -27,8 +27,7 @@ public class SubredditImage
             Url = new Uri(url);
             // Handle legacy image urls
             // http://thumbs.reddit.com/FULLNAME_NUMBER.png
-            int discarded;
```

Comment:
Actually for this specific number this doesn't matter, if wildcards were implemented I would use it as it is only used to see if it's an integer or not, nothing else, so I won't change that